examples/filesystem/README.md

```diff
@@ -1,31 +1,39 @@
 # File system usage example
 
-This is a basic example how to use a file system with RIOT in your embedded application.
+This basic example shows how to use a file system with RIOT in your embedded
+application.
 
-This example shows:
- - how to mount/format/unmount a filesystem, with spiffs, littlefs and constfs examples
- - how to open/read/write/close a file with and without newlib
+In particular, this example shows:
+
+- how to mount/format/unmount a filesystem, with spiffs, littlefs and constfs
+  examples
+- how to open/read/write/close a file with and without newlib
 
 In RIOT, most filesystems use `mtd` as flash interface. So to use this example
-you must define `MTD_0`. `MTD_0` is a pointer to a `mtd_dev_t` instance.
-This example use `littlefs` as default filesystem on the whole `mtd` device.
+one must define `MTD_0`. `MTD_0` is a pointer to a `mtd_dev_t` instance.
+
+This example uses `littlefs` as default filesystem on the whole `mtd` device.
```

Comment:
Here and below, MTD already states for "device" (Memory Technology Device) so using `MTD device` is a pleonasm.